_plugins/redcarpet2_markdown.rb

```diff
@@ -1,48 +0,0 @@
-require 'fileutils'
-require 'digest/md5'
-require 'redcarpet'
-require 'albino'
```

Comment:
Alibno is no longer maintained (https://github.com/github-archive/albino), this is the only thing that uses it, i'm not sure why it's needed. removing this allows me to remove Alibno from our gem file and upgrade to the latest everything